src/packages/engine-core/src/NodeEngine.ts

```diff
@@ -199,6 +199,9 @@ export class NodeEngine {
     this.generator = generator
     this.datasources = datasources
     this.logEmitter = new EventEmitter()
+    this.logEmitter.on('error', () => {
+      // to prevent unhandled error events
```

Comment:
Are the unhandled error events logged somewhere or just completely ignored?

Reply:
Yeah so that's to work around the default behavior of Node.js.
If you have this code:
```ts
const EventEmitter = require('events')

const emitter = new EventEmitter()

emitter.emit('error', new Error('blub'))
```
Node will always throw an unhandled error exception.
However, in our case that's not what we want, as we already handle errors through the promises that are bound to queries. Our error events are just an additional optional way to log them.